js/base/Exchange.js

```diff
@@ -859,6 +859,50 @@ module.exports = class Exchange {
         };
     }
 
+    currencyStructure () {
+        return {
+            'info': undefined,
+            'id': undefined,
+            'lowercaseId': undefined,
+            'uppercaseId': undefined,
+            'numericId': undefined,
+            'code': undefined,
+            'precision': undefined,
+            'type': undefined,
+            'margin': undefined,
+            'name': undefined,
+            'active': undefined,
+            'deposit': undefined,
+            'withdraw': undefined,
+            'fee': undefined,
+            'fees': undefined,
+            'networks': undefined,
+            'address': undefined,
```

Comment:
@kroitor @samgermain we have removed all cases of `address` from currencies, so no need for that field.
```suggestion
```